homeassistant/components/lamarzocco/switch.py

```diff
@@ -0,0 +1,115 @@
+"""Switch platform for La Marzocco espresso machines."""
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from typing import Any
+
+from lmcloud.const import LaMarzoccoModel
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import LaMarzoccoEntity, LaMarzoccoEntityDescription
+from .lm_client import LaMarzoccoClient
+
+
+@dataclass(frozen=True, kw_only=True)
+class LaMarzoccoSwitchEntityDescription(
+    SwitchEntityDescription,
+    LaMarzoccoEntityDescription,
```

Comment:
I think they need to be the other way around

Reply:
doesn't matter apparently, but I'll move them for style